tests/_support/Step/Acceptance/Installation/Installation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Step\Acceptance\Installation;
+
+use Page\Acceptance\Administrator\LoginPage;
+use Page\Acceptance\Administrator\ControlPanelPage;
+
+class Installation extends \AcceptanceTester
+{
+	/**
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		$I = $this;
+		// @todo: the following path should be taken from configuration and not hardcoded as it is now.
+		$I->assertFileExists('tests/joomla-cms3/configuration.php');
```

Comment:
I am not sure if this is a necessary Given at all.. As it is something that is implied for any test is working at all.

Like we also don't write Given that a Earth is existing or Given that we have CPU resources etc.

// cc @javigomez @pritalpatel  What do you think?